frontend/contribute.html

```diff
@@ -147,8 +147,8 @@ <h1 class="page_title ">SUBMIT A DATASET</h1>
 
                         <div class="rodi_section_01 ">
                             <div class="row">
-                            <div class="col-xs-12 col-sm-2 col-md-2 col-lg-2"></div>
-                            <div class="col-xs-12 col-sm-8 col-md-8 col-lg-8 flex-container col">
+
+                            <div class="col-xs-12 col-sm-12 col-md-10 col-md-push-2 col-lg-8 flex-container col">
```

Comment:
Is it intent to change the width of the column? Before, in `sm` breakpoint, its width was of 8 column, pushed by 2 column. Now it's 12 columns with no push.

Reply:
not the width but the position, centered when larger screen, because before removing div empty columns  when go full-screen column not pushed stay always on the left